src/content/structured/components/side-nav/guidance.mdx

```diff
@@ -24,8 +24,6 @@ tabs:
 import {
   IcSideNavigation,
   IcNavigationItem,
-  IcNavigationGroup,
-  IcDivider,
   IcAlert,
 } from "@ukic/react";
 
```

Comment:
Does the date from this file on line 6 now need updating like the code file's has?